### ISSUE TITLE
Issue with youtube

### ISSUE THREAD

**Issue by Mystqr4** [wanhose/cookie-dialog-monster#26]
Since the latest update I have had a few issues with youtube regarding scripts and "Youtube Preview and Ratings" and various popout extensions such as "Youtube Popout Player" on Waterfox G3. each time I popout or preview a youtube video I just get a black screen however after disabling your extension I can now see the videos again and everything works as they once did.

**Comment by wanhose** [comment]
Welcome to our community @Mystqr4!

We can't reproduce this problem, can you add more information such as Waterfox exact version, URLs of extensions that are incompatible with CDM to install them and screenshots (or videos) about this problem?

We don't support natively this browser but we saw that they added support to Chrome extensions a few months ago. Please add the information and we will take a look.

Thank you again for your feedback 😊

**Comment by Mystqr4** [comment]
**- Waterfox version G3.2.6** 
https://cdn.waterfox.net/releases/win64/installer/Waterfox%20G3.2.6%20Setup.exe

**- Cookie Dialog Monster version 5.3.3**

**- FireMonkey version 2.34 (Extension)** 
https://github.com/erosman/support

**- Popup This Video 1.1.5 (Extension)** 
https://github.com/ronsen/popup-this-video

**- Youtube Popout Player version: 4.1.0 (Extension)** 
https://rthaut.github.io/YouTubePopoutPlayer/

**- YouTube Video Preview and Ratings Keyless version: 20210723 (userscript)** 
https://greasyfork.org/en/scripts/397094-youtube-video-preview-and-ratings-keyless

Basically anything on youtube's home page. Without CDM enabled I can usually hover over the video thumbnails and see a preview window alternatively to that I can use a script to show a window to preview/watch the video which also does not work and only shows a black screen. It seems like anything that previews or pops out a video from youtube has this issue.
This problem wasn't present in previous versions of CDM.

Thanks again for the great extension.

**Comment by wanhose** [comment]
Awesome information @Mystqr4, we have planned a release for today with a lot of great stuff related to performance and translations but it won't include this fix. We are going to start researching about this problem and I will keep you updated through this conversation.

Thank you too for using our extension ❤️

**Comment by Mystqr4** [comment]
I will keep testing subsequent releases (including the next one) and report any other issues but so far this extension seems really good. Great work! I look forward to the next release and hopefully seeing a resolve to this bug in the future.

Thanks so much.

**Comment by wanhose** [comment]
Awesome information @Mystqr4, we have planned a release for today with a lot of great stuff related to performance and translations but it won't include this fix. We are going to start researching about this problem and we will keep you updated through this conversation.

Thank you too for using our extension ❤️

**Comment by wanhose** [comment]
Hey again @Mystqr4, can you confirm if this problem persists after updating to **5.4.1**?

**Comment by Mystqr4** [comment]
Excellent! It seems to be working now again.
I will continue to monitor it in the coming days.

Thank you so much for the update.

**Comment by wanhose** [comment]
Well @Mystqr4, I'm going to close the issue. Feel free to reopen this issue if needed.

**Comment by Mystqr4** [comment]
Latest update has sadly caused this problem to return. Tested on Librewolf also.

**Comment by wanhose** [comment]
Hey @Mystqr4,

What a strange behaviour, I didn't change nothing since our last conversation. I'm going to test something locally these days and if it works I will disable definitely the `all_frames` option which I think is responsible.

Thank you for your feedback again and happy new year 😄

**Comment by wanhose** [comment]
Hi @Mystqr4,

Could you update to [5.4.5](https://github.com/wanhose/cookie-dialog-monster/releases/tag/5.4.5) and confirm if everything is working again?

**Comment by Mystqr4** [comment]
Updated and still not working I am afraid. :/

**Comment by wanhose** [comment]
I wasn't able to reproduce this bug @Mystqr4, do you want to have a call to show me how to reproduce it? 🙂

**Comment by Mystqr4** [comment]
I forgot that I could disable the extension per site which did resolve the problem but out of the box it does not.

Here is a gif of issue without the extension enabled and with the extension disabled.
https://imgur.com/a/dzX9hxb

**Comment by wanhose** [comment]
I think you would like the new release (5.4.6). It seems fixed, please force update if it wasn't updated automatically.

**Comment by Mystqr4** [comment]
WOW! You are brilliant!!
Works fantastically well now. 
I really appreciate the effort. This extension is really great and I am glad I found it.
